cmd/docker/docker.go

```diff
@@ -58,6 +82,10 @@ func newDockerCommand(dockerCli *command.DockerCli) *cli.TopLevelCommand {
 	cmd.SetOut(dockerCli.Out())
 	cmd.SetErr(dockerCli.Err())
 
+	// Cobra's context may be nil in some cases, so initialize it here.
```

Comment:
Cobra's context should _never_ be nil.\
A context should never be nil, ever.
Did something change upstream in cobra?
I remember specifically discussing this when contexts were added.

Reply:
Ha! This was from your original PR, but we were discussing this as well in https://github.com/docker/cli/pull/4599#discussion_r1422487271

I can remove this part 👍

Reply:
Ugh... maybe I hit a case where it was nil.

![2RXV3M](https://github.com/docker/cli/assets/799078/1b7e9bdb-ae06-4ea2-be00-5e174c9a10ca)
![2nselq](https://github.com/docker/cli/assets/799078/c55878a3-83cd-4e25-ae4c-09982c4da357)